nrich-registry/src/main/java/net/croz/nrich/registry/data/constant/RegistryDataConstants.java

```diff
@@ -17,20 +17,30 @@
 
 package net.croz.nrich.registry.data.constant;
 
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
 public final class RegistryDataConstants {
 
-    public static final String CREATE_REQUEST_SUFFIX = "%sCreateRequest";
+    public static final String CREATE_REQUEST_CLASS_NAME_FORMAT = "%sCreateRequest";
 
-    public static final String UPDATE_REQUEST_SUFFIX = "%sUpdateRequest";
+    public static final String UPDATE_REQUEST_CLASS_NAME_FORMAT = "%sUpdateRequest";
 
-    public static final String REQUEST_SUFFIX = "%sRequest";
+    public static final String REQUEST_CLASS_NAME_FORMAT = "%sRequest";
 
     public static final String REGISTRY_FORM_ID_FORMAT = "%s:::%s";
 
     public static final String REGISTRY_FORM_ID_CREATE_SUFFIX = "create";
 
     public static final String REGISTRY_FORM_ID_UPDATE_SUFFIX = "update";
 
+    public static final String REQUEST_PACKAGE_NANE = "request";
```

Comment:
There is a typo in the constant name. Additionally, maybe rename this to `REQUEST_CLASS_PACKAGE_NAME` to be consistent with other constants.